core/debug/src/debug.c

```diff
@@ -60,83 +60,6 @@ void debug_init(void)
     SCnSCB->ACTLR |= 0x2;
 }
 
-static void debug_cx_state(int _indent)
```

Comment:
Since it is too much work to do now, tracking in a ticket sounds good to me.

I'd delete the code before the refactor commit, to keep everything building without any awkward commenting out in any commits.